packages/office-ui-fabric-react/src/components/Label/Label.scss

```diff
@@ -15,7 +15,7 @@
   color: $ms-color-neutralPrimary;
   box-sizing: border-box;
   display: block;
-  padding: 5px 0;
+  padding: 8px 0;
```

Comment:
Avoid this change, update individual components and ensure the padding is correct there.

Honestly, a better change would be to be consistent. Other components don't have `top-padding` but they do have `bottom-margin`. Why is Label different?